controllers/books/new.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
       book_title: req.body.title,
       book_author: req.body.author
     })
-    .then(() => res.redirect("/books"))
+    .then((book) => res.redirect(`/usersBooks/${book.id}`))
```

Comment:
Is this just for the post request? Do we need to redirect away from it? (What is on this page?)